src/core/control/Control.cpp

```diff
@@ -2441,12 +2474,12 @@ void Control::quit(bool allowCancel) {
     this->scheduler->removeAllJobs();
     this->scheduler->unlock();
     this->scheduler->stop();  // Finish current task. Must be called to finish pending saves.
-    this->closeDocument();    // Must be done after all jobs has finished (Segfault on save/export)
+    this->discardDocument();  // Must be done after all jobs has finished (Segfault on save/export)
     settings->save();
     g_application_quit(G_APPLICATION(gtkApp));
 }
 
-auto Control::close(const bool allowDestroy, const bool allowCancel) -> bool {
+auto Control::closeDocumentInteractively(const bool allowDestroy, const bool allowCancel) -> bool {
```

Comment:
We could consider renaming that method to close again if we would add something like forceClose as argument.